lib/cs_guide_web/controllers/search_venue_controller.ex

```diff
@@ -7,7 +7,7 @@ defmodule CsGuideWeb.SearchVenueController do
     venues =
       Venue.all()
       |> Venue.preload([:venue_types, :venue_images])
-      |> Enum.sort_by(& &1.venue_name)
+      |> Enum.sort_by(&{5 - &1.cs_score, &1.venue_name} )
```

Comment:
`5 - &1.cs_score` is to sort the score by the highest one first.
I try to add a `sorter` to the `sort_by` function : Enum.sort_by(&{5 - &1.cs_score, &1.venue_name}, &>=/2) )` however the sorter is applied also to `&1.venue_name` (ie z to a) and I don't think there is a way to only apply the sorter to only one of the values.
see https://hexdocs.pm/elixir/Enum.html#sort_by/3